docs/src/App.js

```diff
@@ -4,55 +4,103 @@
 //  found in the LICENSE file in the root directory of this source tree.
 //  You may not use this file except in compliance with the License.
 
-import React from 'react';
+import React, { useState } from 'react';
 import { MDXProvider } from '@mdx-js/tag';
 import { BrowserRouter as Router, Route, Link, Redirect } from 'react-router-dom';
 import routes, { componentList } from './routes';
 import styled from 'styled-components';
-
-const Container = styled.div`
-  display: flex;
-  flex-direction: row;
-`;
-
-const Header = styled.header`
-  flex-direction: column;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding: 1rem;
-  text-align: center;
-`;
+import Logo from './Logo';
 
 const SideBar = styled.aside`
   width: 240px;
   position: fixed;
-  top: 0;
   left: 0;
-  bottom: 0;
-  overflow-y: scroll;
-  padding: 1rem;
-  background-color: #f1f1f1;
-  ul {
+  top: 0;
+  overflow: hidden;
+  @media only screen and (max-width: 680px) {
+    right: 0;
+    width: 100%;
+    background: #1f1e27;
+    z-index: 999;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+`;
+
+const LogoLink = styled(Link)`
+  padding: 2rem;
+  display: block;
+  color: #f7f7f3;
+  &:hover {
+    color: #f7f7f3;
+    opacity: 0.5;
+  }
+`;
+
+const Navigation = styled.ul`
+  max-height: calc(100vh - 88px);
+  overflow-y: auto;
+  padding: 0 2rem 2rem;
+  margin: 0;
+  list-style-type: none;
+  width: 100%;
+  @media only screen and (max-width: 680px) {
+    display: ${(props) => (props.isMobileNavOpen ? 'block' : 'none')};
+  }
+`;
+
+const Section = styled.li`
+  margin-bottom: 1rem;
+  display: block;
+
+  ul,
+  li {
     padding: 0;
     margin: 0;
-    list-style: none;
-    li {
-      a {
-        display: inline-block;
-        font-size: 0.875rem;
-        padding-left: 8px;
-        line-height: 1.2;
-      }
+    list-style-type: none;
+  }
+
+  li {
+    margin-bottom: 0.25rem;
+    a {
+      color: #f7f7f3;
+      font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+      font-size: 14px;
     }
   }
+
+  div {
```

Comment:
Seems like `p` might be a better tag for this (e.g. paragraphs typically have a margin-bottom)?